database/bigtable/bigtable.md

```diff
@@ -0,0 +1,54 @@
+PACKAGE DOCUMENTATION
```

Comment:
Does it this file (database/bigtable/bigtable.md) repeat with APIDocs/bigtable.md ?